### DISCUSSION TITLE
Expose shadowsocks to WAN via reverse proxy (Traefik)

### DISCUSSION THREAD

**Discussion by rubbo898** [qdm12/gluetun#764] [Q&A]
First of all a huge thanks for the great service!

My goal is to configure the proxy so I can use it from the WAN too. 
I'm using Traefik as reverse proxy so my first option is to stay with it but I can evaluate an alternative one if it solves the struggles. 
The idea is to configure the shadowsocks client to connect to gluetun.mydomain.com. I've already tried with no luck, I think the issue is with the UDP traffic or/and a bad Traefik configuration.

If you or anyone else have a working setup (even if not Traefik) to use as example it would be much appreciated! 

Thank you

**Comment by qdm12** [comment]
Hey there!

I use it on my domain.com:8388 without issue. No need for traefik really since Shadowsocks isn't running over http. Just publish the Shadowsocks ports on your Gluetun container and forward the port on your router. You might need to create a DNS record for gluetun.domain.com though if you don't already have one.

Good luck!

**Reply by rubbo898** [reply]
Actually my struggle to make it work with RP was useless... Don't know why my brain was so worried about the idea to open one more port. "KISS" always pay off, it works smooth as you suggested! Thanks
